requirements.txt

```diff
@@ -1,6 +1,7 @@
+# nanopub
```

Comment:
TODO: Once https://github.com/fair-workflows/nanopub/pull/5 is merged and published to pypi we should use that instead of this git install.

Reply:
Yep, and state the specific version to avoid problems that way too. I've reviewed nanopub PR5, so we can probably merge it now.

Reply:
Done